sonar-php-plugin/src/test/java/org/sonar/plugins/php/phpunit/PhpUnitSensorTest.java

```diff
@@ -29,21 +29,16 @@
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.MockitoAnnotations;
-import org.sonar.api.batch.SensorContext;
 import org.sonar.api.batch.fs.FileSystem;
-import org.sonar.api.batch.fs.InputFile;
 import org.sonar.api.batch.fs.internal.DefaultFileSystem;
-import org.sonar.api.batch.fs.internal.DefaultInputFile;
+import org.sonar.api.batch.sensor.SensorContext;
 import org.sonar.api.config.Settings;
-import org.sonar.api.resources.Project;
 import org.sonar.plugins.php.MockUtils;
 import org.sonar.plugins.php.PhpPlugin;
-import org.sonar.plugins.php.api.Php;
 import org.sonar.test.TestUtils;
 
-import static org.fest.assertions.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
@vilchik-elena fest assert is a dead project, assertj is new and already brings stuff, such as the "as" method